ethcore/src/miner/miner.rs

```diff
@@ -1123,6 +1123,9 @@ impl MinerService for Miner {
 		// First update gas limit in transaction queue
 		self.update_gas_limit(chain);
 
+		// Update minimal gas price
+		self.recalibrate_minimal_gas_price();
```

Comment:
Previously it was done in `prepare_block`, so for mining nodes it was the same (`chain_new_blocks` even is not sent to miner if we're syncing afaik)
Also `GasCalibrator` supports a minimal interval between calibrations:
https://github.com/ethcore/parity/blob/8517369372f2e151dedf5d6f1b6c32bd372453da/ethcore/src/miner/miner.rs#L147-L167
so I don't see an issue here.